README.md

```diff
@@ -193,7 +193,7 @@ After cloning the repository the `.env` file needs to be configured. A template
 - `LOG_LEVEL`: 'INFO' or 'DEBUG' recommended
 
 
-### Registering as an RSB Provider
+### Registering as a RSB Provider
```

Comment:
Mega nitpick: This is wrong. The rule deciding between `a`/`an` is whether the **sound** of the word following the article starts with a vowel or consonant, not how its spelled. 

In this case RSB is pronounced `Are-Es-Bee` therefore `an` is correct.

Reply:
Oh wow. Interesting. :scream:
Sorry for that... :expressionless: